server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java

```diff
@@ -408,6 +420,12 @@ public boolean processNonQuery(PhysicalPlan plan)
         return true;
       case SHOW_QUERY_RESOURCE:
         return processShowQueryResource();
+      case MIGRATION:
+        operateMigration((SetMigrationPlan) plan);
```

Comment:
Please rename MIGRATION to SET_MIGRATION and operateMigration to operateSetMigration.

Reply:
Fixed.